### PR TITLE
feat(components): adding multiple seletion behaviour to blip-select

### PR DESCRIPTION
**Pull request by nicolasamaduro**
feature/158104-add-multiple-to-blip-selected

### CODE REVIEW

File: src/components/blipSelect/index.js [hunk 1]
```diff
@@ -463,9 +508,13 @@ export class BlipSelect extends Component {
       case 27: // esc
         if (this.isSelectOpen) {
           this._closeSelect()
-          if (this.input.value && this.selectedOption.label) {
+          if (this._isSomethingSelected()) {
```
Comment (liviaalmeida) [inline src/components/blipSelect/index.js]: A better name would be `isAnySelected`

File: src/scss/components/_select.scss [hunk 2]
```diff
@@ -273,12 +273,12 @@
...
-  .blip-select__option__content {
-    margin-left: 10px;
+.blip-select__option__content {
+  margin-left: 10px;
 
-    span {
-      display: block;
-    }
+  span {
+    display: block;
```
Comment (liviaalmeida) [inline src/scss/components/_select.scss]: If you're using `display: block` on a span, why not just use a `div` then?

Comment (nicolasamaduro) [reply src/scss/components/_select.scss]: I've thought about it, but as long it's legacy code and it's used in the entire portal I've realized that it's wise don't change it for now

File: src/components/blipSelect/index.js (hunk 1 above)
Comment (nicolasamaduro) [reply src/components/blipSelect/index.js]: Done

**Pull request merged**